tests/test_app.py

```diff
@@ -1,14 +1,14 @@
 from unittest import TestCase
-from contextlib import contextmanager
 import json
+import sys
 
-from flask import appcontext_pushed, g
 from werkzeug.datastructures import Headers
 
 import env
+import test_secrets
+sys.modules['api.secrets'] = test_secrets
```

Comment:
I know it's nitpicky, but we should mock this! I can make a PR off of this branch to demonstrate, if that would help.

Reply:
nitpicky is fine! but i would appreciate some guidance here :)

Reply:
Cool! I'll pull something together real quick.